test/src/http/fetchSpec.js

```diff
@@ -112,7 +114,7 @@ describe('Fetch HTTP Backend', () => {
         })
         .then((_response) => {
             expect(_response).to.deep.equal({
-                data: null,
+                data: {},
```

Comment:
I think we should always returns a `null` body if it is empty. Can you fix this and I'll merge you next ?